packages/utils/telemetry-utils/src/sampledTelemetryHelper.ts

```diff
@@ -57,10 +66,12 @@ interface Measurements {
  *
  * @internal
  */
-export class SampledTelemetryHelper implements IDisposable {
+export class SampledTelemetryHelper<TCustomMetrics extends string> implements IDisposable {
```

Comment:
These changes represent a non-trivial feature added to `SampledTelemetryHelper`. Please be sure to document the relevant aspects in the TSDoc comment here.

Let's also be sure to document `TCustomMetrics` with a `@typeParam` comment block.